src/main/java/dev/drugowick/threehundredsixty/domain/entity/Question.java

```diff
@@ -58,5 +60,15 @@ public Question(String category, String title, String description, Employee eval
         this.evaluator = evaluator;
     }
 
+    @JsonIgnore
+    @Formula("CASE" +
+            " WHEN evaluation = 'Não atende' THEN 1" +
+            " WHEN evaluation = 'Atende parcialmente' THEN 2" +
+            " WHEN evaluation = 'Atende' THEN 3" +
+            " WHEN evaluation = 'Excede expectativas' THEN 4" +
+            " ELSE 0 " +
+            "END")
+    private Integer evaluationValue;
```

Comment:
Legal essa anotation do JPA hien! :clap: :clap: 
Pergunta, esse SQL faz ficar preso a um determinado banco (se ela não for 100% ascii, e usar algo específico) ou é agnostico de DB?

Reply:
Massa, né?

Cara, confesso que isso é uma dúvida. Em teoria o CASE é suportado pelo MySQL e H2 que são nossos bancos agora, mas talvez possa ter limitações pra outros BDs pois o @Formula usa SQL não o HQL ou JPQL.